packages/terra-form-fieldset/src/terra-dev-site/test/form-fieldset/FieldsetWithHiddenLegend.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Field from 'terra-form-field';
+import Input from 'terra-form-input';
+import Fieldset from '../../../Fieldset';
+
+const fieldsetwithhiddenlegend = () => (
+  <Fieldset
+    type="checkbox"
```

Comment:
I don't think that having `type`, `name`, `value` and `error` here are adding anything to the use-case.

Reply:
yes we don't need them here It was copy paste mistake, [removed](https://github.com/cerner/terra-core/blob/e1f32ffcf116761ffddcc01deb5d3545ab8febca/packages/terra-form-fieldset/src/terra-dev-site/test/form-fieldset/FieldsetWithHiddenLegend.test.jsx#L7)